### PR TITLE
Update events to always show the time (in EST/EDT)

### PR DESCRIPTION
**Pull request by jakeakopp**
and stop referencing undefined property 'hasTime'.

Also add basic instructions for running the server to the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,10 @@
 # baritonesWebsite
-Website for the Bearded Baritones.  This website is live at http://beardedbaritones.com.  
+Website for the Bearded Baritones.  This website is live at http://beardedbaritones.com.
+
+To run the server:
+1) Install node.js from nodejs.org
```
Comment (proverbialsunrise) [inline README.md]: This should really be a markdown formatted list.  e.g. 

```
# Install node.js from nodejs.org
# Get the secret config file, and place it in `config/index.js`
.
.
.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,10 @@
...
+2) Get the secret config file, and place it in config/index.js
+3) Install the server:
```
Comment (proverbialsunrise) [inline README.md]: This command doesn't really install the server.  It installs the dependencies for the project.

**Comment by jakeakopp** [comment]
@proverbialsunrise I uploaded changes - not sure if you get automatically notified, or what the standard procedure is around that...

**Comment by proverbialsunrise** [comment]
Merged and pushed live.  Glad you commented as I don't think I'm automatically notified of that.

**Pull request merged**